README.md

```diff
@@ -123,20 +123,31 @@ Usage:
-  -e, --exclude string             exclude tests matching this Go regexp (e.g. to exclude all tests beginning with "91", use "91.*").
-                                   If you want more permanent exclusion, check the 'testoverride' option in the config file.
-  -h, --help                       help for run
-      --id string                  (deprecated). Use --include matching your test only.
-  -i, --include string             include only tests matching this Go regexp (e.g. to include only tests beginning with "91", use "91.*").
-      --max-marker-log-lines int   maximum number of lines to search for a marker before aborting (default 500)
-      --max-marker-retries int     maximum number of times the search for log markers will be repeated.
-                                   Each time an additional request is sent to the web server, eventually forcing the log to be flushed (default 20)
-  -o, --output string              output type for ftw tests (default "normal")
-      --read-timeout duration      timeout for receiving responses during test execution (default 1s)
-      --show-failures-only         shows only the results of failed tests
-  -t, --time                       show time spent per test
+      --connect-timeout duration               timeout for connecting to endpoints during test execution (default 3s)
+  -d, --dir string                             recursively find yaml tests in this directory (default ".")
+  -e, --exclude string                         exclude tests matching this Go regexp (e.g. to exclude all tests beginning with "91", use "91.*").
+                                               If you want more permanent exclusion, check the 'testoverride' option in the config file.
+  -f, --file string                            output file path for ftw tests. Prints to standard output by default.
+  -h, --help                                   help for run
+  -i, --include string                         include only tests matching this Go regexp (e.g. to include only tests beginning with "91", use "91.*").
+      --max-marker-log-lines int               maximum number of lines to search for a marker before aborting (default 500)
+      --max-marker-retries int                 maximum number of times the search for log markers will be repeated.
+                                               Each time an additional request is sent to the web server, eventually forcing the log to be flushed (default 20)
+  -o, --output string                          output type for ftw tests. "normal" is the default. (default "normal")
+      --read-timeout duration                  timeout for receiving responses during test execution (default 1s)
+      --show-failures-only                     shows only the results of failed tests
+  -t, --time                                   show time spent per test
+      --wait-delay duration                    Time to wait between retries for all wait operations. (default 1s)
```

Comment:
What is the format for time specifications?

Reply:
Added comment. We already had duration for other flags.